accounting_reporting_core/src/main/java/org/cardanofoundation/lob/app/accounting_reporting_core/domain/entity/TransactionEntity.java

```diff
@@ -86,6 +86,15 @@ public class TransactionEntity extends CommonEntity implements Persistable<Strin
     // https://www.baeldung.com/java-enums-jpa-postgresql
     private LedgerDispatchStatus ledgerDispatchStatus = NOT_DISPATCHED;
 
+    @Embedded
+    @AttributeOverrides({
+            @AttributeOverride(name = "primaryBlockchainType", column = @Column(name = "primary_blockchain_type")),
+            @AttributeOverride(name = "primaryBlockchainHash", column = @Column(name = "primary_blockchain_hash"))
+    })
+    @Getter
+    @Setter
```

Comment:
The whole object here is @Nullable actually, not individual fields. In DB individual fields but in Java, the whole object.